tests/model_registry/model_catalog/test_labels_endpoint.py

```diff
@@ -0,0 +1,95 @@
+import pytest
+import yaml
+
+from kubernetes.dynamic import DynamicClient
+from simple_logger.logger import get_logger
+
+from ocp_resources.config_map import ConfigMap
+from ocp_resources.resource import ResourceEditor
+
+from utilities.infra import get_openshift_token
+from timeout_sampler import TimeoutSampler
+
+from tests.model_registry.model_catalog.utils import (
+    get_labels_from_configmaps,
+    get_labels_from_api,
+    verify_labels_match,
+)
+
+LOGGER = get_logger(name=__name__)
+
+
+class TestLabelsEndpoint:
+    """Test class for the model catalog labels endpoint."""
+
+    @pytest.mark.smoke
```

Comment:
I don't feel this is a candidate for smoke. If the labels are missing, it could be a critical bug, but won't be a blocker for AI Hub. May be change it to sanity?

Reply:
My understanding is that this endpoint is also used by the dashboard team to properly show the catalog UI, hence it would be a bug impacting across teams. IMHO it makes sense to have it at the smoke level.